hello/test/test.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <assert.h>
+#include "../src/hello.h"
+
+int
+main(void) {
+
+  char *foo = hello("foo");
+  char expected[12] = "Hello: foo\n";
```

Comment:
oh... 12.
On Windows, some compilers convert \n to CR+LF (2 bytes).
"Hello: foo" 10 bytes, CR+LF 2 bytes, NULL terminate character 1 byte, the total size is 13 bytes.
Buffer over flow may occur.